bin/wrap.js

```diff
@@ -13,7 +13,9 @@ try {
   include: process.env.NYC_INCLUDE ? process.env.NYC_INCLUDE.split(',') : [],
   enableCache: process.env.NYC_CACHE === 'enable',
   sourceMap: process.env.NYC_SOURCE_MAP === 'enable',
-  instrumenter: process.env.NYC_INSTRUMENTER
+  instrumenter: process.env.NYC_INSTRUMENTER,
+  hookRunInContext: process.env.NYC_HOOK_RUN_IN_CONTEXT === 'true',
```

Comment:
For consistency, I might stick with setting the environment variable to `enable`; I think this approach originated from not wanting to get confused by variable type (`!!'false' === true`).